tests/cases/conformance/types/literal/templateLiteralTypes2.ts

```diff
@@ -2,10 +2,10 @@
 // @declaration: true
 
 function ft1<T extends string>(s: string, n: number, u: 'foo' | 'bar' | 'baz', t: T) {
-    const c1 = `abc${s}`;  // `abc${string}`

```

Comment:
those tests and comments were introduced in https://github.com/microsoft/TypeScript/pull/41891/files#diff-23c99a459cde66a01fe87782c5d76a17b3c75706ad05adc1567eff86995b89e2 but then the change was reverted without removing tests (or adjusting those original comments) here: https://github.com/microsoft/TypeScript/pull/42588